examples/react/css/src/refinement-list.css

```diff
@@ -0,0 +1,45 @@
+/* RefinementList */
```

Comment:
Is this comment needed?

Reply:
I'll delete it, it's a relic from my one-file PoC